packages/mongodb-artifact-generator/src/commands/generateReleaseNotes.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  createConfiguredAction,
+  withConfig,
+  withConfigOptions,
+} from "../withConfig";
+import { createCommand } from "../createCommand";
+import { makeRunLogger, type RunLogger } from "../runlogger";
+import { getReleaseArtifacts } from "../release-notes/getReleaseArtifacts";
+import { summarizeReleaseArtifacts } from "../release-notes/summarizeReleaseArtifacts";
+import {
+  ReleaseArtifact,
+  releaseArtifactShortMetadata,
+} from "../release-notes/projects";
+import { promises as fs } from "fs";
+import YAML from "yaml";
+import { ReleaseInfo } from "../release-notes/ReleaseInfo";
+
+let logger: RunLogger;
+
+type GenerateReleaseNotesCommandArgs = {
+  runId?: string;
+  releaseInfo: string;
+};
+
+export default createCommand<GenerateReleaseNotesCommandArgs>({
+  command: "generateReleaseNotes",
+  builder(args) {
+    return withConfigOptions(args)
+      .option("runId", {
+        type: "string",
+        demandOption: false,
+        description: "A (hopefully unique) name for the run.",
```

Comment:
what happens if not unique?

Reply:
It will use/overwrite the existing `runlogs/<runId>` directory